.github/workflows/testing.yml

```diff
@@ -74,6 +84,16 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Enable webpack persistent caching
+        uses: actions/cache@v3
+        id: build-webpack-persistent-cache
+        with:
+          path: node_modules/.cache
+          key: build-${{ hashFiles('**/webpack.*.js') }}-${{ hashFiles('**/yarn.lock') }}
```

Comment:
I just noticed we don't have any `webpack.*.js` files. All we have are `webpack.*.mjs` files, does it matter?

Reply:
Yes, it should be `.mjs` otherwise key won't change when the webpack config is changed.
```suggestion
          key: build-${{ hashFiles('**/webpack.*.mjs') }}-${{ hashFiles('**/yarn.lock') }}
```

Reply:
I think there're many other `*.js`.

Reply:
In this repo?

Reply:
This pull request, e.g.:

![CleanShot 2022-07-19 at 19 28 55@2x](https://user-images.githubusercontent.com/1091472/179739810-04802fcf-035a-4f59-9a58-6dafc293f2b0.png)

Reply:
Oh yes. Sorry! I got confused.